sale_pricelist_triple_discount/view/pricelist_view.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+        <record model="ir.ui.view" id="product_pricelist_view_form_discount">
+            <field name="name">product.pricelist.item.form.discount</field>
+            <field name="model">product.pricelist.item</field>
+            <field name="inherit_id" ref="product.product_pricelist_item_form_view" />
+            <field name="arch" type="xml">
+                <field name="price_discount" position="after">
+                    <field name="discount2" />
+                    <field name="discount3" />
+                </field>
```

Comment:
I don't see the discount2 and discount3 fields appear anywhere...

I suggest replace this part by this :

```suggestion
            <xpath expr="//field[@name='percent_price']/.." position="after">
                <label for="discount2" attrs="{'invisible':[('compute_price', '!=', 'percentage')]}"/>
                <div attrs="{'invisible':[('compute_price', '!=', 'percentage')]}">
                    <field name="discount2" class="oe_inline" attrs="{'invisible':[('compute_price', '!=', 'percentage')]}"/>
                    %
                </div>
                <label for="discount3" attrs="{'invisible':[('compute_price', '!=', 'percentage')]}"/>
                <div attrs="{'invisible':[('compute_price', '!=', 'percentage')]}">
                    <field name="discount3" class="oe_inline" attrs="{'invisible':[('compute_price', '!=', 'percentage')]}"/>
                    %
                </div>
            </xpath>
```
For this result :
<img width="1142" alt="Capture d’écran 2022-09-11 à 18 12 25" src="https://user-images.githubusercontent.com/19540535/189538277-688031ba-4ebc-4b2f-ba08-959d0b47aba4.png">

Reply:
The fields discount2 and discount3 should appear when you select the 'formula' strategy for price calculation. In this regards the module behave exactly like the version is been migrated from. I've started from this stale PR (https://github.com/OCA/sale-workflow/pull/1643) migrating the module to 12.0, and adapted to the 14.0 which we're interested in targeting.